client-app/config/targets.js

```diff
@@ -1,18 +1,26 @@
-'use strict';
+"use strict";
 
 const browsers = [
-  'last 1 Chrome versions',
-  'last 1 Firefox versions',
-  'last 1 Safari versions'
+  "last 1 Chrome versions",
+  "last 1 Firefox versions",
+  "last 1 Safari versions",
 ];
 
-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
```

Comment:
We're dropping ie11 support here. Obviously fine for Discourse, but since this is a public gem I think we should call it out in the changelog and make it a major version bump. 

(or... maybe just uncomment to restore ie11 support and deal with dropping it later?)